malaria-app-android/src/main/java/com/peacecorps/malaria/fragment/HomeScreenFragment.java

```diff
@@ -473,6 +473,16 @@ public void missedDayRecord(int day, int month, int year){
 
     }
 
+    @Override
+    public void onStop() {

```

Comment:
There is no guarantee that the `onStop` method will be called when we leave an activity. You need to move the code to `onPause` method.
For more info please read - https://developer.android.com/reference/android/app/Activity.html